packages/sources/dlc-btc-por/src/transport/proof-of-reserves.ts

```diff
@@ -210,7 +210,12 @@ export class DLCBTCPorTransport extends SubscriptionTransport<TransportTypes> {
       txId,
       requestConfig,
     )
-    return response.data.result
+
+    const result = response.data.result
+
+    if (!result) console.log('BITCOIN_RPC_URL - getrawtransaction failed: ', response)
```

Comment:
We need to use a `logger` instance

Reply:
Done! went with the logger.error method, but open to a different method.